evm/spec/tables/memory.tex

```diff
@@ -43,8 +43,11 @@ \subsection{Memory}
   \item $e_i (e_i - 1) = 0$,
   \item $(1 - e_i) (a_{i + 1} - a_i) = 0$,
   \item $c_i < 2^{32}$.
+  \item $e_i (v_{i + 1}) < 2^{32}$
```

Comment:
`v` is the value being read or written. I think you should replace `v` by `a` (the address) or something like `a_{virtual}`.